easybuild/easyconfigs/j/Java/Java-1.8.eb

```diff
@@ -16,7 +16,8 @@ toolchain = SYSTEM
 
 dependencies = [
     ('Java', {'arch=x86_64': '%(version)s.0_241',
-              'arch=POWER': '%(version)s_191-b26-OpenJDK'}),
+              'arch=POWER': '%(version)s_191-b26-OpenJDK',
```

Comment:
Hm, will this still work right if the latter part is split into a suffix?

Reply:
Yes, because the `SYSTEM` toolchain is used. So in terms of how dependencies are located, it'll still look for `Java-1.8_191-b26-OpenJDK.eb` (where `version = '1.8_191'` and `versionsuffix = 'b26-OpenJDK`)

Reply:
Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/11545/commits/44f8af7ba2dcd9a87366df32a2563a32e5fd80f9? Maybe?